webpack/webpack.config.babel.js

```diff
@@ -0,0 +1,67 @@
+import autoprefixer from 'autoprefixer';
+import colorLighten from 'less-color-lighten';
+import path from 'path';
+
+function absPath() {
+  let args = [].slice.apply(arguments);
+  args.unshift(__dirname, '..');
+
+  return path.resolve.apply(path.resolve, args);
+}
+
+// Can override this with npm config set dcos-ui:external_plugins ../some/relative/path/to/repo
+let pluginsDir = absPath(process.env.npm_package_config_external_plugins || 'plugins');
```

Comment:
Defaulting to `plugins` provides us with a resolvable directory so we don't get any errors. It won't be used other than to safely resolve `require.context` in `Loader.js`. If pluginsDir is invalid (can only happen if a developer does `npm config set dcos-ui:external_plugins some/dir`) where `some/dir` is invalid then we throw an error because it's not resolvable.